src/pages/NewWarehouse/NewWarehouse.jsx

```diff
@@ -0,0 +1,51 @@
+import "./NewWarehouse.scss";
+import ArrowBack from "../../assets/Icons/arrow_back-24px.svg";
+import FormFields from "../../components/FormFields/FormFields";
+import { Link } from "react-router-dom";
+
+const NewWarehouse = () => {
+
+  const onSubmit = (event) => {
+    event.preventDefault()
```

Comment:
does this reset the input?
`event.target.reset`

Reply:
we can use useState approach on the form and put the func inside onClick and the onChange on the input dom elements @trumerfe @Edthev